moped-database/migrations/1650301717000_alter_project_list_view/up.sql

```diff
@@ -0,0 +1,89 @@
+drop view project_list_view;
+
+CREATE VIEW public.project_list_view AS
+with moped_team_members as (
```

Comment:
The CTE is doing just what it needs to do:

* Create a list of all personnel, per project, with `mpp.status_id` of `1`
* Collect their full names and various roles into an aggregated, comma delimited string of people, for each project
* And make this collection available to the main query below.

Would you consider changing the name of the ephemeral, in memory, table to perhaps `project_person_list_lookup` or something like that? I would love to do something like `project__person_list__lookup` or `project_personList_lookup` -- trying to find a way to show it's `projects` and `person_lists` that are the two things being correlated, not `projects`, `persons`, and `lists`. My suggested name could be improved, i'm sure.

My idea is make it as clear as possible what that construction is doing when reading the joins down below. These never hit the disk nor are available outside the context of this query, so I think dropping the `moped_` prefix is actually helpful and clear here, as well. It differentiates them from the tables on disk.